### PR TITLE
OBPIH-5494 Use user attributes in the Hotjar and enable it on react

### PR DESCRIPTION
**Pull request by awalkowiak**

### CODE REVIEW

File: grails-app/views/common/_hotjar.gsp [hunk 1]
```diff
@@ -11,5 +11,11 @@
         r.src=t+h._hjSettings.hjid+j+h._hjSettings.hjsv;
         a.appendChild(r);
     })(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');
+
+    // Hotjar user attributes
+    // https://help.hotjar.com/hc/en-us/articles/4402892526487-What-are-User-Attributes-
+    // https://help.hotjar.com/hc/en-us/articles/360038394053-How-to-Set-Up-User-Attributes
+    var userId = '${session?.user?.id}' || null;
+    window.hj('identify', userId, {});
```
Comment (awalkowiak) [inline grails-app/views/common/_hotjar.gsp:19]: @jmiranda The `{}` is a custom user attribute config. Do we need to track now something else than only the user id? Examples can be seen here: https://help.hotjar.com/hc/en-us/articles/360038394053-How-to-Set-Up-User-Attributes

File: grails-app/views/common/_hotjar.gsp [hunk 2]
```diff
@@ -11,5 +11,11 @@
         r.src=t+h._hjSettings.hjid+j+h._hjSettings.hjsv;
         a.appendChild(r);
     })(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');
+
+    // Hotjar user attributes
+    // https://help.hotjar.com/hc/en-us/articles/4402892526487-What-are-User-Attributes-
+    // https://help.hotjar.com/hc/en-us/articles/360038394053-How-to-Set-Up-User-Attributes
+    var userId = '${session?.user?.id}' || null;
```
[the remark this answers is not in the document]
Comment (jmiranda) [reply grails-app/views/common/_hotjar.gsp:18]: Yes, username seems like it would work better.

File: grails-app/views/common/_hotjar.gsp (hunk 1 above)
Comment (jmiranda) [reply grails-app/views/common/_hotjar.gsp:19]: At the moment I don't think we know what we'll need. Let's take it one step at a time though. Once we get the username set we'll be able to do some testing to see what else would be useful.

**Review by jmiranda — CHANGES_REQUESTED** [review]
See comments

**Pull request merged**